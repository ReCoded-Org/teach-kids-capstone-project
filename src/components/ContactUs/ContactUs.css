@media only screen and (max-width: 600px) {
  .ContactUs .contactInfo  {
    font-size: 13px;
    margin-left: 30px;
    max-width: 400px;
  }
  .FormTextContactUs{
      max-width: 290px;
  }
  
  
}