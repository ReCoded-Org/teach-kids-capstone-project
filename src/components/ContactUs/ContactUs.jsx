import React from "react";
import Phone from "../../assets/Phone.png";
import Message from "../../assets/Message.png";
import Location from "../../assets/Location.png";
 
function ContactUs() {
    return (
      <div className="px-12 sm:px-12 lg:px-28 flex flex-col md:flex-col lg:flex-row md:justify-around ">
          <div className="pt-32 md:mr-48">
          <div className="col-md-6 col-sm-12">
            <div className="col">
            <h1 className="text-5xl font-bold font-quicksand text-blue-dark mb-10">Contact Us</h1>
              <p className="text-lg text-justify font-SourceSansPro text-blue-dark mb-10 md:max-w-lg">Get the understanding you need to channel the books of spells you have always been smitten with, and boost sales which cartoon rabbits love to steal! Get the understanding you need to</p>
                <div className="flex pb-20 ">
                  <div className="w-10 mr-5  grow text-blue-dark h-5 font-SourceSansPro">
                    <img src={Location} alt="Location icon" />
                    <p>9094 Bay Meadows Street Conyers, GA 30012</p>
                    </div>
                    <div className="flex-initial text-blue-dark w-8 grow h-5 font-SourceSansPro">
                      <img src={Phone} alt="Phone icon" />
                      <p>+229-955-5388-336</p>
                      <p>+229-955-5373-360</p>
                      </div>
                      <div className="flex-initial ml-5 w-8 grow font-SourceSansPro">
                        <img src={Message} alt="Contact icon" />
                        <p>support@mail.com</p>
                        <p>office@mail.com</p>
                        </div>
                      </div>
                      </div>
                      </div>
          </div>
          <div className="lg:w-5/12 my-8 rounded-tr">
              <form id="contact" className="bg-gray-800 rounded-tr font-SourceSansPro">
                  <div className="justify-between">
                          <div className="flex flex-col">
                              <label className="text-lg text-blue-dark font-semibold mb-2 ">Full Name</label>
                              <input required id="full_name" name="full_name" type="text" className="focus:border focus:border-blue-dark rounded border-none bg-gray mb-6" placeholder="Full Name"/>
                          </div>
                      </div>
                          <div className="flex flex-col">
                          <label className="text-lg text-blue-dark font-semibold mb-2">Email</label>
                              <input required id="email" name="email" type="email" className="focus:border focus:border-blue-dark rounded border-none bg-gray mb-6" placeholder="example@email.com"/>
                          </div>
                      <div className="flex flex-col">
                      <label className="text-lg text-blue-dark font-semibold mb-2">Message</label>
                          <textarea placeholder="Type Here" name="message" className="border-none bg-gray mb-6 rounded" rows="9" id="message"/>
                      </div>
                      <button type="submit" className="rounded-md border-2 border-blue-dark text-white p-2 font-SourceSansPro w-56 font-semibold bg-blue-dark  duration-300 ease-linear hover:border-blue-dark hover:bg-white hover:text-blue-dark hover:shadow">Send</button>
              </form>
          </div>
      </div>
    )
}
 
export default ContactUs;
