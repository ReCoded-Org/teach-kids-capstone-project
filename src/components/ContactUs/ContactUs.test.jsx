import renderer from "react-test-renderer";
import ContactUs from "../ContactUs/ContactUs";

it("renders correctly", () => {
    const tree = renderer.create(<ContactUs />).toJSON();
    expect(tree).toMatchSnapshot();
});