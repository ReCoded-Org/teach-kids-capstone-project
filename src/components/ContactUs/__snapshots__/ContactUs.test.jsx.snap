// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="px-12 sm:px-12 lg:px-28 flex flex-col md:flex-col lg:flex-row md:justify-around "
>
  <div
    className="pt-32 md:mr-48"
  >
    <div
      className="col-md-6 col-sm-12"
    >
      <div
        className="col"
      >
        <h1
          className="text-5xl font-bold font-quicksand text-blue-dark mb-10"
        >
          Contact Us
        </h1>
        <p
          className="text-lg text-justify font-SourceSansPro text-blue-dark mb-10 md:max-w-lg"
        >
          Get the understanding you need to channel the books of spells you have always been smitten with, and boost sales which cartoon rabbits love to steal! Get the understanding you need to
        </p>
        <div
          className="flex pb-20 "
        >
          <div
            className="w-10 mr-5  grow text-blue-dark h-5 font-SourceSansPro"
          >
            <img
              alt="Location icon"
              src="Location.png"
            />
            <p>
              9094 Bay Meadows Street Conyers, GA 30012
            </p>
          </div>
          <div
            className="flex-initial text-blue-dark w-8 grow h-5 font-SourceSansPro"
          >
            <img
              alt="Phone icon"
              src="Phone.png"
            />
            <p>
              +229-955-5388-336
            </p>
            <p>
              +229-955-5373-360
            </p>
          </div>
          <div
            className="flex-initial ml-5 w-8 grow font-SourceSansPro"
          >
            <img
              alt="Contact icon"
              src="Message.png"
            />
            <p>
              support@mail.com
            </p>
            <p>
              office@mail.com
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="lg:w-5/12 my-8 rounded-tr"
  >
    <form
      className="bg-gray-800 rounded-tr font-SourceSansPro"
      id="contact"
    >
      <div
        className="justify-between"
      >
        <div
          className="flex flex-col"
        >
          <label
            className="text-lg text-blue-dark font-semibold mb-2 "
          >
            Full Name
          </label>
          <input
            className="focus:border focus:border-blue-dark rounded border-none bg-gray mb-6"
            id="full_name"
            name="full_name"
            placeholder="Full Name"
            required={true}
            type="text"
          />
        </div>
      </div>
      <div
        className="flex flex-col"
      >
        <label
          className="text-lg text-blue-dark font-semibold mb-2"
        >
          Email
        </label>
        <input
          className="focus:border focus:border-blue-dark rounded border-none bg-gray mb-6"
          id="email"
          name="email"
          placeholder="example@email.com"
          required={true}
          type="email"
        />
      </div>
      <div
        className="flex flex-col"
      >
        <label
          className="text-lg text-blue-dark font-semibold mb-2"
        >
          Message
        </label>
        <textarea
          className="border-none bg-gray mb-6 rounded"
          id="message"
          name="message"
          placeholder="Type Here"
          rows="9"
        />
      </div>
      <button
        className="rounded-md border-2 border-blue-dark text-white p-2 font-SourceSansPro w-56 font-semibold bg-blue-dark  duration-300 ease-linear hover:border-blue-dark hover:bg-white hover:text-blue-dark hover:shadow"
        type="submit"
      >
        Send
      </button>
    </form>
  </div>
</div>
`;
